nomad/structs/csi.go

```diff
@@ -117,33 +117,33 @@ func (t *TaskCSIPluginConfig) Copy() *TaskCSIPluginConfig {
 // CSIVolumeCapability is the requested attachment and access mode for a
 // volume
 type CSIVolumeCapability struct {
-	AttachmentMode CSIVolumeAttachmentMode
-	AccessMode     CSIVolumeAccessMode
+	AttachmentMode VolumeAttachmentMode
+	AccessMode     VolumeAccessMode
 }
 
-// CSIVolumeAttachmentMode chooses the type of storage api that will be used to
+// VolumeAttachmentMode chooses the type of storage api that will be used to
 // interact with the device.
-type CSIVolumeAttachmentMode string
+type VolumeAttachmentMode string
```

Comment:
slightly unintuitive that this now-generalized thing still lives in `csi.go`

Reply:
Good point. I'll move this into `nomad/structs/volumes.go`. Looking at that file, I see that the `VolumeRequest` validation is also using the "CSI" values for validating host volumes too, so I'll tweak that.

Reply:
Done!